rust/perspective-viewer/src/rust/components/column_settings_sidebar/sidebar.rs

```diff
@@ -132,7 +136,8 @@ pub fn ColumnSettingsSidebar(p: &ColumnSettingsProps) -> Html {
                             { renderer }
                             { custom_events }
                             { selected_column }
-                            { on_close }/>
+                            { on_close }
+                            />
```

Comment:
We need to sync up on style guide for `html!` macro content, because it is immune to `rustfmt` and changes like these make it difficult for me to use file history to bisect issues.